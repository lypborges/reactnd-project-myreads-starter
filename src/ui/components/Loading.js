import React from "react";
import { Spin } from "antd";

export default () => (
  <div className="list-spinner">
    <Spin size="large" />
  </div>
);
