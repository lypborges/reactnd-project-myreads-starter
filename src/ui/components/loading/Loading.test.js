import React from "react";
import Loading from "./Loading";
import renderer from "react-test-renderer";

describe("Loading component layout", () => {
  test("render", () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
