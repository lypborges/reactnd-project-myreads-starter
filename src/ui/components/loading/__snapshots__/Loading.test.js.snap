// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component layout render 1`] = `
<div
  className="list-spinner"
>
  <div
    className="ant-spin ant-spin-lg ant-spin-spinning"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i />
      <i />
      <i />
      <i />
    </span>
  </div>
</div>
`;
